group/iab/IAB_Decisions.md

```diff
@@ -54,66 +58,104 @@ for the IAB chair to determine whether it will be held open or secret.
+   by the IAB.
+
+2. Ensure that all IAB members have an opportunity to discuss the topic or
+   question in some meeting of the IAB.
+
+3. Formally start the vote, following the discussion(s).
+
+4. Conclude the vote once votes from all IAB members have been collected,
+   or once 7 days have elapsed and there are sufficient votes received
+   to meet the requirements in RFC 2850.
+
+Detailed guidelines for each of these steps are provided below.
+
+### Announcing votes
+
+Announcements of votes are sent over email, but can also be provided over
+other channels. These announcements will be sent by the IAB or the executive director.
+
+Announcements should be as specific as possible about the question on which
+the vote will be taken, and background material should be provided or already be
+available to the IAB.
+
+Announcements can optionally indicate that ballots will be secret.
+
+Announcements should include the timeframe for the vote, including the
+opportunities for discussion (see below) and when the vote is expected to start.
+
+### Discussion prior to voting
+
+IAB members should have the opportunity to discuss topics before taking a vote.
+This discussion can occur in both asynchronous (email, messages) and synchronous
+(teleconferences, in-person meetings) forms.
+
+Once a vote is announced, the period for asynchronous discussion starts immediately.
+This discsssion generally takes place on a private email list amongst voting
+members.
+
+The discussion period also should include opportunities for each voting IAB member
+to have synchronous discussions with other voting members. Depending on the exact
+meeting schedule of the IAB, this requirement can be satisfied in different ways.
+For example, an in-person meeting at an IETF meeting or an IAB teleconference where
+all IAB members are expected to attend satisfies this requirement in a single
+discussion. Alternatively, this can be satisfied by having multiple discussions that
+occur at different times to accomodate IAB members in different timezones.
+Note that lack of attendence by IAB members due to vacations or choosing not to
+attend a meeting will not delay being able to hold the vote, as long as meeting
+opportunities were provided.
+
+Any meeting on which a vote is discussed needs to either be a regularly scheduled
+meeting, or be an extraordinary meeting scheduled at least a week in advance.
+
+Discussions in meetings need to follow a regular procedure for adding agenda items
+to the meeting. Voting discussions should be added to a meeting agenda at least
+48 hours prior to the meeting, or earlier depending on the current meeting practice
+of the IAB.
+
+### Conducting votes
+
+Once the discussion opportunities (described above) have been provided, a vote
+can formally start. This can occur during a teleconference or in-person meeting
+directly after discussion, but unless all IAB members are present, will always
+also be started virtually.
```

Comment:
```suggestion
also be continued virtually.
```

Reply:
Suggestion to make it clear that the vote can be started in the IAB meeting following the discussion, and continue on via email for missing votes.